pachyderm/pachyderm
# Lbliii/pachctl doc updates

This PR updates the cobra version, modifies `pachctl-doc` to output within its directory (previously it was still building a path for mkdocs), and re-writes CLI descriptions/examples for all PFS commands.

## How to Test the Pachctl CLI's docs output

1. Navigate to `/src/server/cmd/pachctl` and run `go build`. 
2. Execute `./pachctl` to see the updated CLI and interact with it.
3. Explore other command outputs in the same way you normally would, for example: `./pachctl list`, `./pachctl put file`, etc.
